pom.xml

```diff
@@ -89,7 +88,7 @@
       <dependency>
         <groupId>io.jenkins.tools.bom</groupId>
         <artifactId>bom-2.138.x</artifactId>
```

Comment:
```suggestion
        <artifactId>bom-2.222.x</artifactId>
```

Reply:
At first, I didn't want to bump the minimum core version require for the plugin but I agree it would be nice.